engine/apps/mobile_app/backend.py

```diff
@@ -33,6 +34,10 @@ def unlink_user(self, user):
         token = MobileAppAuthToken.objects.get(user=user)
         token.delete()
 
+        # delete push notification related info for user
+        APNSDevice.objects.filter(user=user).delete()
```

Comment:
This is unrelated to FCM, this line just makes sure that when user is unlinked, their push notification info is deleted as well as the mobile auth token.